src/main/java/org/folio/service/manager/input/InputDataManagerImpl.java

```diff
@@ -223,6 +180,19 @@ protected ExportManager getExportManager() {
     return vertx.getOrCreateContext().get(ExportManager.class.getName());
   }
 
+  private Future<FileDefinition> findFileDefinition(String fileDefinitionId, String tenantId) {
```

Comment:
Let's move this method to FileDefinitionService as you make the same refactoring for job execution